optaplanner-core/src/main/java/org/optaplanner/core/impl/solver/DefaultSolver.java

```diff
@@ -179,11 +182,13 @@ public void solvingStarted(SolverScope<Solution_> solverScope) {
         super.solvingStarted(solverScope);
         int startingSolverCount = solverScope.getStartingSolverCount() + 1;
         solverScope.setStartingSolverCount(startingSolverCount);
-        logger.info("Solving {}: time spent ({}), best score ({}), environment mode ({}), random ({}).",
+        logger.info("Solving {}: time spent ({}), best score ({}), move thread count ({}), " +
+                "environment mode ({}), random ({}).",
```

Comment:
1) We show the environmentMode in both the "solving started" and "solving ended" log.

(soft suggestion) How about we show the move thread count in both too?

Reply:
2. (very very soft suggestion, wdyt? your call) I might mention environment mode before move thread count because it's more important when they're looking at score calculation speed, as it distorts env mode even more than move thread count.